scripts/stdshaders/buildshaders.bat

```diff
@@ -87,7 +87,7 @@ goto end
 echo - ERROR: ShaderCompile.exe doesn't exist in devtools\bin
 goto end
 
-:NoShaderCompile
+:NoShaderCrc
```

Comment:
Actually the whole ShaderCrc.exe check should be removed, as evererything is now in single exe